src/gui/optionsdlg.cpp

```diff
@@ -439,6 +440,12 @@ void OptionsDialog::loadWindowState()
     const Preferences* const pref = Preferences::instance();
 
     resize(pref->getPrefSize(this->size()));
+}
+
+void OptionsDialog::loadWindowStateAfter()
```

Comment:
maybe `loadSplitterState`?

Reply:
and please add a short description why this must be called after `show()`.

Reply:
done

Reply:
comment added